levels/lovelace_tower/objectives/api-01-local-function/description.md

```diff
@@ -0,0 +1,29 @@
+# Local Function
+
+<div class="aside">
+<h3>Requirements</h3>
+<ul>
+  <li>Create a function called `wordSwapper`.</li>
+  <li>This function receives 3 string arguments.</li>
+  <li>Replace all instances of the "target" string in the "source" string with the "replacement" string, using the `replaceAll` function.</li>
+  <li>Return the resulting string.</li>
+  <li>Once you're done, press <em>HACK</em>.</li>
+</ul>
+</div>
+
+Before we jump into working with Web APIs let's try interacting with a function locally. For this, we'll use the `String.replaceAll` method to replace all instances of a particular string with something else.
```

Comment:
```suggestion
Before we jump into working with Web APIs let's remind ourselves about local APIs provided by libraries, as functions and objects. For this, we'll use the `String.replaceAll` method to replace all instances of a particular string with something else.
```

Minor suggestion to make the "libraries expose APIs too" link explicit

Reply:
I like this! It really hits home on the idea that APIs extend beyond the Web!